dll/win32/userenv/userenv.rc

```diff
@@ -122,4 +122,7 @@ LANGUAGE LANG_NEUTRAL, SUBLANG_NEUTRAL
 #endif
 #ifdef LANGUAGE_ZH_CN
     #include "lang/zh-CN.rc"
+#endif
+#ifdef LANGUAGE_ZH_TW
+    #include "lang/zh-TW.rc"
 #endif
```

Comment:
Add newline at end of file.

Reply:
Ah yes, there's a missing one in that file.